### ISSUE TITLE
LightningLens checkpoints are storing model weights as well as lens weights

### ISSUE THREAD

**Issue by 123epsilon** [msakarvadia/AttentionLens#40]
This is bloating the size of the lens and is unnecessary unless I'm missing something

**Comment by msakarvadia** [comment]
Good catch, you are not missing anything (I think it was just an overlooked detail). We can save only the lens weights moving forward.
